includes/functions.php

```diff
@@ -7,22 +7,28 @@ function confirm_query($result_set) {
 		}
 	}
 	
-	function get_all_subjects() {
+	function get_all_subjects($public = true) {
```

Comment:
Ideally for all function we should provide a description.
for  [example](https://d.pr/i/ZC2jIG)

Reply:
got it, I'll make sure not to repeat this again, thanks !